test/functionalTest/cases/0000_json_parse/jsonParsePostUpdateContextSubscription.test

```diff
@@ -28,8 +28,15 @@ dbInit CB
 brokerStart CB
```

Comment:
Just to check: is this PR including any "real" change in .test?

I have seen some improvements (e.g. typo fixing, steps preamble in comments, attribute value changes in order to align them with step number) but not sure if is there anything else appart fom that.